flink-runtime/src/main/java/org/apache/flink/runtime/security/token/DefaultDelegationTokenManagerFactory.java

```diff
@@ -34,37 +31,21 @@
 import java.io.IOException;
 import java.util.concurrent.ExecutorService;
 
-/** A factory for {@link KerberosDelegationTokenManager}. */
+/** A factory for {@link DefaultDelegationTokenManager}. */
 @Internal
-public class KerberosDelegationTokenManagerFactory {
+public class DefaultDelegationTokenManagerFactory {
 
     private static final Logger LOG =
-            LoggerFactory.getLogger(KerberosDelegationTokenManagerFactory.class);
+            LoggerFactory.getLogger(DefaultDelegationTokenManagerFactory.class);
```

Comment:
I think this is not used anymore. In some future PR you might want to remove this, and the imports for it.